### PR TITLE
Rename initialize_const_values_to_ops and const_values_to_ops to initialize_ops_to_simplifiers and ops_to_simplifiers

### PR DESCRIPTION
**Pull request by Krovatkin**

### CODE REVIEW

File: src/ngraph/pass/algebraic_simplification.cpp [hunk 1]
```diff
@@ -241,8 +241,8 @@ bool ngraph::pass::AlgebraicSimplification::run_on_function(std::shared_ptr<ngra
         }
 
         const Node& node = *n;
-        auto eh = ops_to_const_values.find(TI(node));
-        if (eh == ops_to_const_values.end())
+        auto eh = ops_to_simplifiers.find(TI(node));
```
Comment (diyessi) [inline src/ngraph/pass/algebraic_simplification.cpp]: Does this want to be `auto&`?

Comment (Krovatkin) [reply src/ngraph/pass/algebraic_simplification.cpp]: `const Node&` -> `auto&`?

Comment (diyessi) [reply src/ngraph/pass/algebraic_simplification.cpp]: No, `auto& eh = ...`
I'm not sure what the find is returning.

Comment (Krovatkin) [reply src/ngraph/pass/algebraic_simplification.cpp]: @diyessi could you please approve this one when you have a moment?

**Pull request merged**